### ISSUE TITLE
Bug when using Bundle in Contao 4.10

### ISSUE THREAD

**Issue by simonla82** [Tastaturberuf/contao-image-copyright-bundle#2]
### Updating existing Contao 4.9 installation to 4.10
- When updating a Contao 4.9 instance using the copyright bundle to Contao 4.10 the Install Tool proposes to Drop the columns `ic_copyright`, `ic_href` and `ic_hide`. Applying this would cause the loose of all inserted copyright information.
![grafik](https://user-images.githubusercontent.com/73082071/103311414-98514480-4a1a-11eb-8023-50bd6c2c8468.png)
- The module "Copyright-Liste" keeps displaying the correct information which has already been inserted.
- There is no possibility to insert new copyright information or edit existing in the file structure.
### Installing the bundle on a clean Contao 4.10 installation
- Installing the bundle via the conato-manager in a clean Contao 4.10 installation works flawless, but the Install Tool does not propose to update the database afterwards.
- There are no input fields for copyright information in the file structure.
- When using the "Copyright-Liste" module, the following error is occurs in the logs: 
```
request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\Exception\InvalidFieldNameException: "An exception occurred while executing 'SELECT * FROM tl_files WHERE ic_copyright != '' AND type = 'file' AND ic_hide = '' AND extension IN ('jpg','jpeg','gif','png','tif','tiff','bmp','svg','svgz','webp')':  SQLSTATE[42S22]: Column not found: 1054 Unknown column 'ic_copyright' in 'where clause'" at /var/www/html/contao/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php line 79 {"exception":"[object] (Doctrine\\DBAL\\Exception\\InvalidFieldNameException(code: 0): An exception occurred while executing 'SELECT * FROM tl_files WHERE ic_copyright != '' AND type = 'file' AND ic_hide = '' AND extension IN ('jpg','jpeg','gif','png','tif','tiff','bmp','svg','svgz','webp')':\n\nSQLSTATE[42S22]: Column not found: 1054 Unknown column 'ic_copyright' in 'where clause' at /var/www/html/contao/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php:79, Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 42S22): SQLSTATE[42S22]: Column not found: 1054 Unknown column 'ic_copyright' in 'where clause' at /var/www/html/contao/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDO/Exception.php:18, PDOException(code: 42S22): SQLSTATE[42S22]: Column not found: 1054 Unknown column 'ic_copyright' in 'where clause' at /var/www/html/contao/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:125)"} []
``` 

## How to reproduce 
Install the bundle on a clean Contao 4.10 installation or update an existing Contao 4.9 installation to 4.10.

## System information
- Contao 4.10.5
- PHP 7.4.13 (cli)
- tastaturberuf/contao-image-copyright-bundle v1.0.4

Any help is highly appreciated - thanks in advance!

**Comment by simonla82** [comment]
### Update
Even weirder: When setting up a clean Contao 4.9 installation, the corresponding columns are created but the input fields for the copyright information are not displayed in the file structure nevertheless. I thought this had worked a few days ago... 

**System information:**
- Contao 4.9.0
- PHP 7.4.13 (cli)
- tastaturberuf/contao-image-copyright-bundle v1.0.4

**Comment by Tastaturberuf** [comment]
Its a bug in Contao https://github.com/contao/contao/issues/2570

I remove the priority and release an new bug fix version in an minute. thanks for the detailed bug report.

**Comment by Tastaturberuf** [comment]
Fixed in 1.0.5

**Comment by Tastaturberuf** [comment]
Thanks @simonla82

**Comment by simonla82** [comment]
> 
> 
> Fixed in 1.0.5

Works flawlessly! Thank you! 🎉
